### ISSUE TITLE
Activation time differences

### ISSUE THREAD

**Issue by pva8** [firemodels/fds#8344]
Hi, I  am using FDS 6.7.3 and inserted several temperature devices with a setpoint of 74 degrees. The .csv file has a time step of about 3 s. Form the user guide I saw that the given output will be time-averaged. However, when I look at the .out file, there is a big difference in activation times, even 100s difference. 
In case I wanted to simulate the sprinkler activation time (by inserting a heat detector), should I take  the activation time given in the .out file, or is it better to use the .csv file?

**Comment by mcgratta** [comment]
100 s difference is clearly not right. Can you post a simple case that demonstrates this problem.

**Comment by pva8** [comment]
I have placed temperature devices as follows:
&DEVC ID='THCP00', QUANTITY='TEMPERATURE', XYZ=10.0,12.0,13.6, SETPOINT=74.0/
Simulation time is 2600 s.  In the .out file the step size is around 0.004 s, while the devc.csv file the temperature is recorded every 2.6 s. I have rechecked the activation times, and there is one with a difference of 214s (see attachment). The first time the temperature of this device gets close to the setpoint temperature is at 236 s (71 degrees), so with still 118.5 s difference compared to the .out file. Before that the temperature of this device stays at least 10 degrees lower than the setpoint temperature.
Could the issue be that the time step difference between the .out and the devc.csv is too big? Should I solve this by adding TIME_AVERAGED=.FALSE. to the DEVC line?
[Activation times.xlsx](https://github.com/firemodels/fds/files/4673748/Activation.times.xlsx)
